src/codewhisperer/util/editorContext.ts

```diff
@@ -219,20 +219,21 @@ function logSupplementalContext(supplementalContext: CodeWhispererSupplementalCo
         return
     }
 
-    getLogger().verbose(`
+    getLogger().debug(`CodeWhispererSupplementalContext:
             isUtg: ${supplementalContext.isUtg},
             isProcessTimeout: ${supplementalContext.isProcessTimeout},
             contentsLength: ${supplementalContext.contentsLength},
             latency: ${supplementalContext.latency},
         `)
 
+    let logString = ''
     supplementalContext.supplementalContextItems.forEach((context, index) => {
-        getLogger().verbose(`
-                -----------------------------------------------
-                Path: ${context.filePath}
-                Score: ${context.score}
-                Chunk: ${index}:${context.content}
-                -----------------------------------------------
-            `)
+        logString += `Chunk ${index}:
+        Path: ${context.filePath}
+        Content: ${index}:${context.content}
+        Score: ${context.score}
+        -----------------------------------------------`
     })
+
+    getLogger().debug(logString)
```

Comment:
This is an improvement, thanks. But there is still a disconnected `getLogger().debug()` call on line 222. Only one one message is needed, not multiple disconnected ones. That also has the advantage of putting a single timestamp on the message.

Reply:
yea make sense, let me update it

Reply:
knowledge learned thanks Justin